sql-to-dbsp-compiler/lib/sqllib/src/casts.rs

```diff
@@ -202,7 +202,7 @@ pub fn cast_to_TimeN_time(value: Time) -> Option<Time> {
 /////////// cast to decimal
 
 #[inline]
-pub fn cast_to_decimal_b(value: bool, precision: u32, scale: i32) -> Decimal {
+pub fn cast_to_decimal_b(value: bool, precision: u32, scale: u32) -> Decimal {
```

Comment:
Did I comment about possibly supporting negative scales in the future?

Reply:
You did comment about it in the previous PR. 
I don't think we should support negative scales. MySQL and Spark don't support it, only Postgres does. And the SQL standard says that "The scale is a non-negative integer."

Reply:
ok